presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -241,7 +240,6 @@
     private final boolean alwaysRevokeMemory;
     private final NodeSpillConfig nodeSpillConfig;
     private final NodeSchedulerConfig nodeSchedulerConfig;
-    private boolean printPlan;
```

Comment:
I would leave it, this is a debug utility. 

I am not sure if it is not used by someone.

@dain you added this in 2014, do still think this is helpful? I have never used this.